### ISSUE TITLE
Can i stop animation before ending point?

### ISSUE THREAD

**Issue by ninmorfeo** [weepy/jquery.path#32]
Hi i created animation of unfolding cards along a one bezier curve.
Can i stop animation before ending's point?

I would need to position the cards like a folding fan

**Comment by weepy** [comment]
can you just use jquery methods of stopping animation ?

**Comment by ninmorfeo** [comment]
Yes i can but I can't understand where to enter the property detection of .square left property.
this is the function:

$ (". square"). each (function () {
for (var i in Paths [type]) {
$ (this) .stop (). animate ({path: Paths [type] [i]}, 1000);

}

**Comment by ninmorfeo** [comment]
ok i find it:
is right ?

  $(".quadrato").each(function () {
                for (var i in Paths[type]) {
                    $(this).stop().animate({path: Paths[type][i]}, {
                        duration: 5000,
                        step: function (now, fx) {
                            console.log($(this).css('left'));                            
                           var pos_corrente=parseInt($(this).css('left'), 10);                            
                            if(pos_corrente>500) {
                                 $(this).stop();
                            }
                        }
                    });
